apps/agent/app/src/components/Balances.js

```diff
@@ -63,18 +14,19 @@ function useBalanceItems(balances) {
   const convertRates = useConvertRates(verifiedSymbols)
 
   const balanceItems = useMemo(() => {
-    return balances.map(({ address, amount, decimals, symbol, verified }) => ({
-      address,
-      amount,
-      convertedAmount: convertRates[symbol]
-        ? amount.divn(convertRates[symbol])
-        : new BN(-1),
-      decimals,
-      symbol,
-      verified,
-    }))
+    return balances.map(({ address, amount, decimals, symbol, verified }) => {
```

Comment:
Small nitpick: leaving the implicit return (`({`) would have made these changes smaller :).